boot/freeldr/freeldr/lib/fs/ext.c

```diff
@@ -0,0 +1,1433 @@
+/*
```

Comment:
Just to be sure -- because git messed up the "file move" and instead understood it as a file deletion + new file -- , this is the ext2.c file renamed, with the "Ext2" --> "Ext" naming changes + your extra Ext4 support?